recipes/libcufft/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+mkdir -p ${PREFIX}/lib
```

Comment:
CI was failing with:
```
mv: cannot move ‘pkg-config’ to ‘/home/conda/staged-recipes/build_artifacts/cufft_1675252566075/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/pkgconfig’: No such file or directory
```
I think after moving the `requirements` (although I am not quite sure).